### PR TITLE
[#28] Integrate dropdown question

### LINKED ISSUE #28: [Integrate] As a user, I can see a dropdown question
## Why
Dropdown questions are one of the most common question types to allow users to select one answer. The dropdown options could be anything, such as, a set of cities, locations, or even yes/no, since they are text-based options.
Even though this is called a dropdown, to provide the mobile experience, we will use a native picker wheel for this type of question.

## Acceptance Criteria
When a Question model has `display_type` as `dropdown`,
- Display a dropdown's picker wheel in the Answer section
- Apply the answers as picker wheel items
- Use the answer's text to show as the picker wheel item's title
- By default, the first answer is initially selected

## Design
[Dropdown Question](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge-Mobile-App?page-id=0%3A1&node-id=48%3A0&scaling=min-zoom)

### PR DESCRIPTION
**Pull request by kaungkhantsoe**
Closes #28 

## What happened 👀

Dropdown questions are one of the most common question types to allow users to select one answer. The dropdown options could be anything, such as, a set of cities, locations, or even yes/no, since they are text-based options.
Even though this is called a dropdown, to provide the mobile experience, we will use a native picker wheel for this type of question.

When a Question model has display_type as dropdown,
- Display a dropdown menu in the Answer section
- Apply the answers as picker items
- Use the answer's text to show as the picker item's title
- By default, the first answer is initially selected

## Insight 📝

- Integrate answers data to the picker menu
- Set the first item as selected by default

**KNOW ISSUE: `DropdownMenu` uses `Column` to show `DropdownMenuItem` instead of `LazyColumn`. Large list of dropdown menus affects the application's responsiveness.**

## Proof Of Work 📹

https://user-images.githubusercontent.com/32578035/190360344-5ca83251-c1ef-43ba-bbe7-ff242d34e53f.mp4

### CODE REVIEW

**Comment by ryan-conway** [comment]
@kaungkhantsoe conflicts 🦀

**Review by AVI5HEK — APPROVED** [review]
LGTM 🚀

File: app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt [hunk 1]
```diff
@@ -57,5 +65,7 @@ fun SurveyQuestionScreenPreview() {
         ),
         pageNumber = 1,
         totalNumberOfPage = 5
-    )
+    ) { _, _ ->
+        // Do nothing
+    }
```
Comment (ryan-conway) [inline app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt:70]: [Optional] We could leave this as a single line, no biggie 😉 
```suggestion
    ) { _, _ -> }
```

**Review by ryan-conway — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**